### DISCUSSION TITLE
Why is BT-142 winner chosen used in 38.EFX while it is not defined in de notice type 38?

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#592] [Q&A] [answered]
The winner is not listed in the contract modifucation notice. It is used in the view template 38.EFX. 
But is not in the noticetype 38. So i think it will be never in the xml and so the winner will not be shown in the contract modification publication .. i am wrong, or is it a bug?

![image](https://github.com/OP-TED/eForms-SDK/assets/111063502/7661979e-fe04-4910-b84f-bf86dc98fa62)

**Comment by pdonohoe** [comment, marked as answer]
Thank you for pointing this out @kletersteegp. This is indeed a bug. It will be fixed with the release of SDK 1.9, and the fix will be applied to previous versions of the SDK.
